File: impex/extends/item_group.py
import frappe
from erpnext.setup.doctype.item_group.item_group import ItemGroup
from impex.impex.doctype.price_change.price_change import add_auto_price_rules

class CustomItemGroup(ItemGroup):
	def on_update(self):
		super(CustomItemGroup, self).on_update()
		add_auto_price_rules("Item Group", self.name)

File: impex/public/js/item_group.js
{% include 'impex/public/js/rule_prices.js' %}

frappe.ui.form.on("Item Group", {
    refresh(frm) {
        if (!frm.is_new()) {
            impex.rule_prices.setup(frm, { html_field: 'custom_rule_prices_html' });
        }
    }
});